lib/backport_session.js

```diff
@@ -61,7 +61,7 @@ export default class BackportSession extends Session {
         cli.stopSpinner(`${file} does not exist in current working tree`,
           cli.SPINNER_STATUS.WARN);
         continue;
-      };
+      }
```

Comment:
Unrelated style change? (I'm surprised we don't lint these files.)

Reply:
(Although, yeah, that semicolon shouldn't be there. Maybe leave it and ignore my comment. But uh, future enhancement for someone: Add linting or adjust the current linting to catch that stuff automatically.)